### DISCUSSION TITLE
Array annotations

### DISCUSSION THREAD

**Discussion by knash94** [openai-php/client#399] [General]
Hey,

I've been encountering significant challenges with the array annotations in this repository. I'm wondering if there's a more efficient method to generate these, or if we might consider a different approach altogether. Addressing this issue could save a lot of time and reduce merge conflicts.

These are a couple of suggestions which I think may help
1. Code Style Enhancements: Introduce a line length limit to our code style guidelines to allow arrays to wrap onto new lines. This change could improve readability and reduce the complexity of resolving merge conflicts.
2. PHP 8 Attributes: Leverage the attributes feature in PHP 8 to enhance reusability. By doing this, we can avoid updating multiple files for a single schema change, streamlining the process and reducing the potential for errors.

**Comment by gehrisandro** [comment]
Hi @knash94 

1. I don't think line breaks are valid here. Or at least PHPStan is not happy with them.
2. Can you give me a concrete example what it would look like?
